src/routing/Api.ts

```diff
@@ -162,14 +163,17 @@ export async function info() {
 export async function routeWithAlternativeRoutes(requestId: number, args: RoutingArgs) {
     const request: RoutingRequest = {
         vehicle: args.vehicle || 'car',
-        elevation: false,
+        elevation: true,
         debug: false,
         instructions: true,
         locale: 'en',
         optimize: 'false',
-        points_encoded: true,
+        // todonow: apparently point decoding does not handle elevation yet
+        points_encoded: false,
         'alternative_route.max_paths': 3,
         algorithm: 'alternative_route',
+        // todonow: simply hard-coded for now
+        details: ['road_environment', 'surface', 'average_speed'] as any,
```

Comment:
Not sure yet how path details shall be chosen via the UI

Reply:
This is fine until there is an options panel where we could put this (but not sure if this should be configurable at all)

(maybe add road_class and max_speed ?)

Reply:
I think sooner or later there definitely should be some way to configure this via the UI (at least some hidden/expert kind of option). But sure for now I will just add the other two encoded values you mentioned.